x-pack/plugins/synthetics/public/apps/synthetics/state/monitor_list/selectors.ts

```diff
@@ -16,8 +16,8 @@ export const selectEncryptedSyntheticsSavedMonitors = createSelector(
   selectMonitorListState,
   (state) =>
     state?.data.monitors.map((monitor) => ({
-      ...monitor.attributes,
-      id: monitor.attributes[ConfigKey.MONITOR_QUERY_ID],
+      ...monitor,
+      id: monitor[ConfigKey.MONITOR_QUERY_ID],
```

Comment:
This line can probably be omitted, as ConfigKey.MONITOR_QUERY_ID = `id`, so this line is no longer doing anything.

Reply:
570e9fa55f3613e27889d067278f4b880d648aca